internal/helpers.go

```diff
@@ -484,8 +484,9 @@ func TemporaryRepo(t *testing.T, w *databricks.WorkspaceClient) string {
 func GetNodeTypeId(env string) string {
 	if env == "gcp" {
 		return "n1-standard-4"
-	} else if env == "aws" {
-		return "i3.xlarge"
+	} else if env == "azure" {
+		return "Standard_DS4_v2"
 	}
-	return "Standard_DS4_v2"
+	// We default to AWS because our "aws-prod-ucws" test environment has CLOUD_ENV set to "ucws"
```

Comment:
Why can't we change it to `aws` instead of `ucws`?

Reply:
We rely on the "ucws" string at multiple places across multiple projects (tf, sdks). It's less of a lift to change the default here.